css/styles.css

```diff
@@ -23,6 +21,11 @@
   object-fit: cover;
 }
 
+img {
+  width: 250px;
+  height: 300px;
+}
+
```

Comment:
What if you use a different image with a different aspect ratio? This would make the image look very skewed. I would just make the image the full width of its containing `column-half` class element.